kebechet/managers/info/info.py

```diff
@@ -35,19 +35,19 @@
 class InfoManager(ManagerBase):
     """Manager for submitting information about running Kebechet instance."""
 
-    def run(self) -> typing.Optional[dict]:
+    def run(self) -> typing.Optional[dict]:  # type: ignore
```

Comment:
Why ignore?

Reply:
Seems like there is an incompatible type with the base class, any suggestion?
`kebechet/managers/info/info.py:38: error: Signature of "run" incompatible with supertype "ManagerBase"`

Reply:
Change `ManagerBase.run( )` and other managers to return `typing.Optional[typing.Dict[Any,Any]]` and see if that fixes the issue

Reply:
Still seems to throw - `Signature of "run" incompatible with supertype "ManagerBase"`